Formula/qt.rb

```diff
@@ -4,8 +4,7 @@ class Qt < Formula
   desc "Cross-platform application and UI framework"
   homepage "https://www.qt.io/"
   url "https://download.qt.io/official_releases/qt/5.13/5.13.2/single/qt-everywhere-src-5.13.2.tar.xz"
-  mirror "http://qt.mirror.constant.com/archive/qt/5.13/5.13.2/single/qt-everywhere-src-5.13.2.tar.xz"
-  mirror "http://qt.mirrors.tds.net/qt/archive/qt/5.13/5.13.2/single/qt-everywhere-src-5.13.2.tar.xz"
+  mirror "https://mirrors.dotsrc.org/qtproject/archive/qt/5.13/5.13.1/single/qt-everywhere-src-5.13.1.tar.xz"
```

Comment:
This appears to not be the same version as the others.

Reply:
Oops, thanks for spotting this! The fix is on the way.